### ISSUE TITLE
Crash when refreshing on MacOSX 10.11.6

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/SourcetrailBugTracker#390]
Steps to reproduce:

- Launch Sourcetrail
- Open a project
- Press Ctrl+R > Crash

or 

- Launch Sourctrail
- Click on the main UI window (not the recent project selector)
- Press Ctrl+R > Crash

crash report:
```
Process:               Sourcetrail [551]
Path:                  /Applications/Sourcetrail.app/Contents/MacOS/Sourcetrail
Identifier:            com.sourcetrail
Version:               ??? (0.12-25-g8f809e4)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Sourcetrail [551]
User ID:               501

Date/Time:             2017-05-22 14:54:15.913 +0200
OS Version:            Mac OS X 10.11.6 (15G1421)
Report Version:        11
Anonymous UUID:        0FD4FC06-304D-3994-4E2F-969A74CD15BB


Time Awake Since Boot: 1300 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
abort() called

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff98eedf06 __pthread_kill + 10
1   libsystem_pthread.dylib       	0x00007fff8bbed4ec pthread_kill + 90
2   libsystem_c.dylib             	0x00007fff91f686df abort + 129
3   org.qt-project.QtCore         	0x000000010707d8c9 0x107067000 + 92361
4   org.qt-project.QtCore         	0x000000010707f297 QMessageLogger::fatal(char const*, ...) const + 231
5   org.qt-project.QtCore         	0x000000010707a13c qt_assert(char const*, char const*, int) + 76
6   com.sourcetrail               	0x00000001046550bb QtGraphicsView::keyPressEvent(QKeyEvent*) + 379
7   org.qt-project.QtWidgets      	0x00000001064bac5a QWidget::event(QEvent*) + 4666
8   org.qt-project.QtWidgets      	0x00000001065b43dd QFrame::event(QEvent*) + 45
9   org.qt-project.QtWidgets      	0x0000000106643049 QAbstractScrollArea::event(QEvent*) + 1769
10  org.qt-project.QtWidgets      	0x00000001067da6f6 QGraphicsView::event(QEvent*) + 214
11  org.qt-project.QtWidgets      	0x000000010647958d QApplicationPrivate::notify_helper(QObject*, QEvent*) + 269
12  org.qt-project.QtWidgets      	0x000000010647b78b QApplication::notify(QObject*, QEvent*) + 4043
13  org.qt-project.QtCore         	0x0000000107253544 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 164
14  org.qt-project.QtWidgets      	0x00000001064d9a0f 0x106468000 + 465423
15  org.qt-project.QtWidgets      	0x000000010647958d QApplicationPrivate::notify_helper(QObject*, QEvent*) + 269
16  org.qt-project.QtWidgets      	0x000000010647bed2 QApplication::notify(QObject*, QEvent*) + 5906
17  org.qt-project.QtCore         	0x0000000107253544 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 164
18  org.qt-project.QtGui          	0x0000000106b9cd13 QGuiApplicationPrivate::processKeyEvent(QWindowSystemInterfacePrivate::KeyEvent*) + 163
19  org.qt-project.QtGui          	0x0000000106b8349b QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 155
20  org.qt-project.QtGui          	0x0000000106b8254c QWindowSystemInterface::flushWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 572
21  libqcocoa.dylib               	0x000000010a333ca3 0x10a313000 + 134307
22  libqcocoa.dylib               	0x000000010a333dc8 0x10a313000 + 134600
23  com.apple.AppKit              	0x00007fff8ccadb15 -[NSWindow _reallySendEvent:isDelayedEvent:] + 2108
24  com.apple.AppKit              	0x00007fff8c6ec539 -[NSWindow sendEvent:] + 517
25  libqcocoa.dylib               	0x000000010a3220e7 0x10a313000 + 61671
26  libqcocoa.dylib               	0x000000010a3219c8 0x10a313000 + 59848
27  com.apple.AppKit              	0x00007fff8c66d16a -[NSApplication sendEvent:] + 4382
28  libqcocoa.dylib               	0x000000010a33cd5e 0x10a313000 + 171358
29  com.apple.AppKit              	0x00007fff8c4d3df2 -[NSApplication run] + 796
30  libqcocoa.dylib               	0x000000010a3396bf 0x10a313000 + 157375
31  org.qt-project.QtCore         	0x000000010724f781 QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 401
32  org.qt-project.QtCore         	0x0000000107253bb5 QCoreApplication::exec() + 341
33  com.sourcetrail               	0x0000000104615b7b QtApplication::exec() + 155
34  com.sourcetrail               	0x000000010460aad4 main + 3428
35  libdyld.dylib                 	0x00007fff8e99d5ad start + 1
```

**Comment by egraether** [comment]
Can reproduce now, is related to #67

**Comment by egraether** [comment]
Fixed with 0.13.12
#67 is still present
